src/DatabaseMain.py

```diff
@@ -0,0 +1,29 @@
+import src.InitializeDatabaseScript as DB
+
+
+if __name__ == "__main__":
+    db = DB.Database()
+    user_name = "postgres"
+    password = "my_secret_password"
+    host = "localhost"
+    port = 5432
+    database_name = "test_db5"
+    table_name = "bug_table"
+    record = ("bug_title", "bug_description", "bug_traceback_info", False)
+    db.connect()
+    db.create_database()
```

Comment:
We should move this connect and create to the main file, where we do first when building the application

Reply:
Done